kolibri_explore_plugin/assets/src/customApps.js

```diff
@@ -141,7 +137,6 @@ export const EkIguanaSections = [
   {
     name: 'Literature and Language Arts',
     channels: [
-      '59bb2e5a3d2e5e3b85b87d9ab4daa2f3', // reading
```

Comment:
Removing channels from EkIguanaSections means nothing to current Endless Key. EkIguanaSections was for the hardcoded order of channels in the landing page of the USB option (which still has the channels in the USB stick). A real cleanup would be to remove EkIguanaSections altogether with the corresponding views/components:
- kolibri_explore_plugin/assets/src/components/EkIguanaList.vue
- kolibri_explore_plugin/assets/src/views/DiscoveryPageEkIguana.vue